docs/content/preview/explore/change-data-capture/cdcsdk-server.md

```diff
@@ -0,0 +1,302 @@
+debezium.source.table.include.list=public.test
+debezium.source.database.master.addresses=127.0.0.1:7100
+debezium.source.snapshot.mode=never
+```
+
+The `debezium.source` configurations are nothing but the Debezium Connector's configurations only where you can specify the `debezium.source` as a prefix to any of the connector's configurations. For a complete list of the Debezium configurations, see [Debezium Connector for YugabyteDB configurations](../change-data-capture/debezium-connector-yugabytedb.md#connector-configuration-properties).
+
+<!-- TODO Vaibhav: add more configuration examples -->
+
+#### Configuring using environment variables
+
+Configuration using environment variables maybe useful when running in containers. The rule of thumb is to convert the keys to UPPER CASE and replace `.` with `_`. For example, `debezium.source.database.port` has to be changed to `DEBEZIUM_SOURCE_DATABASE_PORT`.
+
+#### Server configuration
+
+| Property | Default | Description |
+| :--- | :--- | :--- |
+| `cdcsdk.server.transforms` | | Transformations to apply. | <!-- TODO: add the complete list of transforms available -->
+
+**Additional configuration:**
+| Property | Default | Description |
+| :--- | :--- | :--- |
+| `quarkus.http.port` | 8080 | The port on which CDCSDK Server exposes Microprofile Health endpoint and other exposed status information. |
+| `quarkus.log.level` | INFO | The default log level for every log category. |
+| `quarkus.log.console.json` | true | Determine whether to enable the JSON console formatting extension, which disables "normal" console formatting. | 
+
+
+#### HTTP Client
+
+The HTTP Client will stream changes to any HTTP Server for additional processing with the original design goal to make Debezium act as a native event source.
+
+| Property | Default | Description |
+| :---- | :---- | :---- |
+| `debezium.sink.type` | | Must be set to `http` |
+| `debezium.sink.http.url` | | The HTTP Server URL to stream events to. This can also be set by defining the K_SINK environment variable, which is used by the Knative source framework. |
+| `debezium.sink.http.timeout.ms` | 60000 | The number of milli-seconds to wait for a response from the server before timing out. |
+
+#### Amazon S3
+
+The Amazon S3 Sink streams changes to an AWS S3 bucket. Only Inserts are supported. The available configuration options are:
+
+| Property | Default | Description |
+| :--- | :--- | :--- |
+| `cdcsdk.sink.type` | | Must be set to `s3`. |
+| `cdcsdk.sink.s3.bucket.name` | | Name of S3 bucket. |
+| `cdcsdk.sink.s3.region` | | Name of the region of the S3 bucket. |
+| `cdcsdk.sink.s3.basedir` | | Base directory or path where the data has to be stored. |
+| `cdcsdk.sink.s3.pattern` | | Pattern to generate paths (sub-directory and filename) for data files. |
+| `cdcsdk.sink.s3.flushBytesMB` | 200 | Trigger Data File Rollover on file size. |
+| `cdcsdk.sink.s3.flushRecords` | 10000 | Trigger Data File Rolloever on number of records |
+
```

Comment:
Add a note for single table support.

Amazon S3 Sink supports a single table at a time. Specifically `cdc.source.table.include.list` should contain only one table at a time. If multiple tables need to be exported to Amazon S3, multiple CDCSDK servers that read from the same CDC Stream ID but write to different S3 locations should be setup.